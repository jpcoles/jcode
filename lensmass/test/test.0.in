#BEGIN LENS dim 1 omega 0 lambda 0
#BEGIN MODEL dim 3
1
2
3
#END MODEL
#END LENS
