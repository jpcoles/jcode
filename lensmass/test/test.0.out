#BEGIN MATRIX dim 3 omega 0.000000 lambda 0.000000
1.000000
2.000000
3.000000
2.000000
4.000000
6.000000
3.000000
6.000000
9.000000
#END MATRIX
#BEGIN EIGENVALUES dim 3 omega 0.000000 lambda 0.000000
14.000001907348632812500000000000000000000000000000000000000000 0.000000000000000000000000000000000000000000000000000000000000
-0.000000220078533175183110870420932769775390625000000000000000 0.000000000000000000000000000000000000000000000000000000000000
0.000000000000000000000000000000000000000000000000000000000000 0.000000000000000000000000000000000000000000000000000000000000
#END EIGENVALUES
